server/src/api.rs

```diff
@@ -155,13 +149,13 @@ impl Game for GameAPI {
         Result<Response<Player>, Status> {
             let player_uuid = parse_uuid_or_fail!(request.into_inner().id);
 
-            match self.core
+            let player = self.core
                 .lock().await
                 .players.lock().await
-                .internal_players.get(&player_uuid) {
-                    Some(player) => Ok(Response::new(player.clone())),
-                    None => return Ok(Response::new(Player::default()))
-                }
+                .internal_players.get(&player_uuid)
+                .map_or(Player::default(), |player| player.clone());
```

Comment:
if `Player::default()` is expensive, it should probably use `map_or_else` instead